packages/attribute-registry-process/test/vitest.api.setup.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { vi } from "vitest";
+import { Request, Response, NextFunction } from "express";
+
+vi.mock("pagopa-interop-application-audit", async () => ({
+  applicationAuditBeginMiddleware: vi.fn(
+    async () => (_req: Request, _res: Response, next: NextFunction) => next()
+  ),
+  applicationAuditEndMiddleware: vi.fn(
+    async () => (_req: Request, _res: Response, next: NextFunction) => next()
+  ),
+}));
+
+vi.mock("pagopa-interop-commons", async () => {
+  const actual = await vi.importActual<typeof import("pagopa-interop-commons")>(
+    "pagopa-interop-commons"
+  );
+  return {
+    ...actual,
+    authenticationMiddleware: vi.fn(
+      () =>
+        async (
+          req: Request,
```

Comment:
Doing this you should not need the cast to any below
```suggestion
          req: Request & { ctx: AppContext },
```